app/assets/javascripts/participants.js.coffee

```diff
@@ -60,3 +60,8 @@ $ ->
   $("#participants-list-table").bootstrapTable 'refresh', {silent: true}
   $("#participants-tracker-table").bootstrapTable 'refresh', {silent: true}
   $("#participant-info").bootstrapTable 'refresh', {silent: true}
+
+(exports ? this).visitSorter = (a, b) ->
```

Comment:
This method name is a little ambiguous. It looks like a comparison function rather than a sorter.